test/client/test.ts

```diff
@@ -1,9 +1,11 @@
-import * as assert from "assert";
-import * as fetch from "node-fetch";
+import assert from "assert";
 import { Crypto } from "@peculiar/webcrypto";
 import { cryptoProvider } from "@peculiar/x509";
 import { ApiClient } from "@peculiar/acme-client";
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const fetch = require("node-fetch");
```

Comment:
I suppose to use node-fetch@^2.x.x rather than use require with comments at each place.

Reply:
It requires `@types/node-fetch` module. It's fixed now